### ISSUE TITLE
http or https for canonical URIs of Getty, Wiki*, and others?

### ISSUE THREAD

**Issue by beaudet** [linked-art/linked.art#577]
We say here: https://linked.art/api/1.0/protocol/

that https is the preferred protocol for Linked Art implementations and presumably that means also for the canonical URIs of published entities.

Getty is still reporting http:// at the top of their concept pages, e.g.: http://vocab.getty.edu/aat/300311458

Will referencing the AAT with https:// create problems with linking up data sets due to the URI's scheme differences?  I see that Yale is using http for both Getty and Wikidata.  Wikimedia, by the way, might have officially switched theirs to https.   So, I guess my question for those familiar with processing data sets from multiple institutions is whether this is a problem that is so common that it must be solved by any system consuming linked data or should implementations pay close attention to the examples given by the authorities when those examples are present?  Or maybe so long as an http -> https redirect is in place at the authority, either will work?

**Comment by beaudet** [comment]
I think wikidata recommends https but I think that's from a security perspective, not necessarily a change in the scheme of canonical URIs. 

It looks like RDK's permalink's use https

https://rkd.nl/artists/10024

**Comment by azaroth42** [comment]
Wikidata asserts their namespace as:

```
@prefix wd: <http://www.wikidata.org/entity/> .
```

in the RDF serializations.

e.g. (and I don't recommend this as it's LONG ... don't say I didn't warn you ...)

`curl -L -H "Accept: text/n3" http://www.wikidata.org/entity/Q42`

(You can `| head -35` to grab the prefixes off the top)

**Comment by edwardanderson** [comment]
It seems like the "Semantic View" URIs for AAT resources has recently (?) switched from `http` to `https`.

**Comment by azaroth42** [comment]
@edwardanderson True, but in the representation it's still http. Eg: https://vocab.getty.edu/aat/300194222.jsonld 
The subject of this representation served via https is `http://vocab.getty.edu/aat/300194222`

It's a pervasive and super frustrating issue :(  And compounded by people using the human view rather than the canonical URI in the data (e.g. aat/page/300194222 instead of aat/300194222) 

In terms of "solve at the right level" this is a data concern, and we could create some helpful tooling around it to fix URIs in bad data (e.g. consider: https://github.com/project-lux/data-pipeline/blob/main/pipeline/config.py#L168-L234 ) and document what we expect ... but data is what it is, and all we can do is whack the moles when they show up.

**Comment by azaroth42** [comment]
@workergnome Let's chat next week -- and we could have that chat in public (again) too if you want, given the topic of the meeting :)

**Comment by beaudet** [comment]
sounds informative. where do I get tickets?

**Comment by azaroth42** [comment]
Is there an issue here? At best a recipe / note that lists the namespaces?
